doh-server/doh-server.conf

```diff
@@ -40,7 +41,7 @@ timeout = 10
 # Number of tries if upstream DNS fails
 tries = 3
 
-# Only use TCP for DNS query
+# Only use TCP for upstream DNS queries
 tcp_only = false
```

Comment:
I guess `tcp_only` can be removed now because we have better options now.

Reply:
Blunder, it was actually already removed in the Go code, I forgot to remove it from here. Done.